service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentsValidatorService.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.fpl.service;
```

Comment:
Maybe move the validation services to it's own package within the service package, just to tidy it up and group some common services